lib/adskHydraSceneBrowser/lib/CMakeLists.txt

```diff
@@ -140,10 +153,17 @@ target_include_directories(${TARGET_NAME}
 # this avoids overlap between Qt macros and boost, and enforces using Q_ macros.
 set_target_properties(Qt6::Core PROPERTIES INTERFACE_COMPILE_DEFINITIONS QT_NO_KEYWORDS)
 
-target_compile_definitions(${TARGET_NAME} 
-    PRIVATE 
-        HDUI_EXPORT
-)
+if(PXR_VERSION VERSION_LESS 2408)
+	target_compile_definitions(${TARGET_NAME} 
+		PRIVATE 
+			HDUI_EXPORT
+	)
+else()
+	target_compile_definitions(${TARGET_NAME} 
+		PRIVATE 
+			HDUI_EXPORTS
+	)
+endif()
```

Comment:
Cleaner to use a generator expression, e.g.
https://github.com/Autodesk/maya-hydra/blob/2a49b976652f2f90eaa00fcd3e4fb0d8285c01da/cmake/compiler_config.cmake#L52

Reply:
Thanks, I'll do it in my next PR as this one has already been merged. The modified code which works fine with usd 24.08 and 24.05 is :
target_compile_definitions(${TARGET_NAME} 
	PRIVATE 
	$<IF:$<VERSION_LESS:${PXR_VERSION},2408>,HDUI_EXPORT,HDUI_EXPORTS>
)